src/Data/CosmosDb/Console.RecruitSeedDataWriter/Data/CandidateSkills.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "_id": "CandidateSkills",
-  "Skills": [
+  "column1Skills": [
```

Comment:
Could this document not just be as it was.  don't think the database representation should know anything about how it is to be presented in the UI. Maybe you can split the list into two lists in the orchestrator?